packages/client/mutations/AcceptTeamInvitationMutation.ts

```diff
@@ -40,7 +40,6 @@ graphql`
     # this is just for the user that accepted the invitation
     team {
       ...DashNavListTeam
-      ...TopBarMeetingsActiveMeetings
```

Comment:
@mattkrick I yanked the TopBarMeetings component and all the places where the fragment was used. I’m not exactly sure the logic behind reusing the fragments everywhere, did we want to update active meetings when these events occurred, like accepting an invite to a new team that may have active meetings? I did not replicate this with the MeetingsDash active meetings fragment. Does this break expected behaviors?